src/Cms/Layouts.php

```diff
@@ -78,21 +78,23 @@ public static function parse($input): array
     /**
      * Converts layouts to blocks
      *
-     * @return Blocks
+     * @return \Kirby\Cms\Blocks
      */
     public function toBlocks()
     {
-        if ($this->isEmpty()) {
-            return Blocks::factory([]);
-        }
-        foreach ($this as $layout) {
-            foreach ($layout->columns() as $column) {
-                foreach ($column->blocks() as $block) {
-                    $blocks[] = $block->toArray();
+        $blocks = [];
+
+        if ($this->isNotEmpty() === true) {
+            foreach ($this->data() as $layout) {
+                foreach ($layout->columns() as $column) {
+                    foreach ($column->blocks() as $block) {
+                        $blocks[] = $block->toArray();
+                    }
                 }
             }
         }
 
         return Blocks::factory($blocks);
     }
+    }
```

Comment:
Oh my mistake on suggestion lines, sorry. You can fix that with removing this line and squash them.

Reply:
Thank you @texnixe for fixing that. Could you squash all commits to single?